docs/ops.html

```diff
@@ -361,14 +351,12 @@ <h4 class="anchor-heading"><a id="basic_ops_increase_replication_factor" class="
   "partitions":[{"topic":"foo","partition":0,"replicas":[5]}]}
 
   Save this to use as the --reassignment-json-file option during rollback
-  Successfully started reassignment of partitions
-  {"version":1,
-  "partitions":[{"topic":"foo","partition":0,"replicas":[5,6,7]}]}</code></pre>
+  Successfully started partition reassignment for foo-0</code></pre>
```

Comment:
Because there is only one topicPartition, output `reassignment` instead of `reassignments`